resources/help/lua_timers.md

```diff
@@ -5,47 +5,47 @@ provided duration between each call.
 
 ## Creating a Timer
 
-***blight:add_timer(secs, repeat, callback)***
+***timer:add(secs, repeat, callback)***
```

Comment:
All the new timer module functions should use timer.<function> instead of timer:<method>.

See comments below.

Reply:
Done in the updated commit.